source/beforereport_04_others.rst

```diff
@@ -73,12 +73,106 @@ Pythonの可能性、カンファレンスの可能性、人と人の交流か
 今回のロゴ制作の過程はまさに「Everyone's different, all are wonderful」という今年のテーマそのものでした。
 
 
-Webサイトについて
+PyCon JP Webサイトいついて
 ------------------------------
 
-hack-a-thon(肉会)について
+こんにちわ!!
+メディアチームの橋本です。
+昨年に引き続き、今年もWebシステムを担当いたしました。
+基本的作業フローや基本的なシステム構築は昨年と同様のものを用いました。
+
+・Symposionを使用したサイト開発
+
+・機能要望のチケット化
+
+・TiDDによるイテレーション開発
+
+詳細に関しましては、 `『PyCon JP 2015の作り方』 <http://gihyo.jp/dev/serial/01/pycon-jp-2015/0001>`_  を参照いただければと思います。
```

Comment:
NGなら先方から指摘来るでしょう。
メディアとしては競合だけど、参照NGとか技術系でやってたら成り立たないっすよー

On Mon, Aug 8, 2016 at 10:32 PM yukofeb <notifications@github.com> wrote:

> In source/beforereport_04_others.rst
> <https://github.com/pyconjp/shoeisha2016/pull/19#discussion_r73872151>:
>
> >  ------------------------------
> >
> > -hack-a-thon(肉会)について
> > +こんにちわ!!
> > +メディアチームの橋本です。
> > +昨年に引き続き、今年もWebシステムを担当いたしました。
> > +基本的作業フローや基本的なシステム構築は昨年と同様のものを用いました。
> > +
> > +・Symposionを使用したサイト開発
> > +
> > +・機能要望のチケット化
> > +
> > +・TiDDによるイテレーション開発
> > +
> > +詳細に関しましては、 `『PyCon JP 2015の作り方』 <http://gihyo.jp/dev/serial/01/pycon-jp-2015/0001>`_  を参照いただければと思います。
>
> @yellow844 <https://github.com/yellow844> @shimizukawa
> <https://github.com/shimizukawa> 大丈夫ですかねー。じゃここはそのままいきたいと思います！
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pyconjp/shoeisha2016/pull/19/files/3f402d76db8d953a631516842057e2a06ac19df5#r73872151>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJQR1I4fqJD8chrETfIaRn68E_L6FWWks5qdyuAgaJpZM4JegmG>
> .
>